src/esp/assets/ResourceManager.cpp

```diff
@@ -1584,12 +1598,16 @@ gfx::PhongMaterialData::uptr ResourceManager::buildFlatShadedMaterialData(
   } else {
     finalMaterial->ambientColor = material.ambientColor();
   }
+
+  finalMaterial->importName = importName;
```

Comment:
We want to give users access to the "import name"--the material name in the source gltf file.